src/components/EventHandler.tsx

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
Also, @fanwgwg has suggested a nice thing about auto-formatting our `librarie.js` codes, can you try it out? *VS Code* has a nice built in way of auto-formatting the current source file, on Mac it is `Shift + Option + F` not sure what is it on Windows.